.github/workflows/test_ert.yml

```diff
@@ -31,6 +31,13 @@ jobs:
           pyproject.toml
           dev-requirements.txt
 
+    - name: Install OPM Flow
```

Comment:
This adds 26 seconds extra to the pipeline.

Reply:
As discussed, this can go away and instead do the testing when doing bleeding builds.

Reply:
Now removed.